tests/pc/test_po_update.py

```diff
@@ -163,7 +165,9 @@ def test_po_update_io_no_loss(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
     # be like:[("10.0.0.56", "10.0.0.57", "PortChannel0001", ["Ethernet48", "Ethernet52"])]
     pcs = [(pair[0], pair[1], pair[2], mg_facts["minigraph_portchannels"][pair[2]]["members"]) for pair in
            peer_ip_pc_pair
-           if len(mg_facts["minigraph_portchannels"][pair[2]]["members"]) >= 2]
+           if len(mg_facts["minigraph_portchannels"][pair[2]]["members"]) >= 2 and
```

Comment:
this is not required if minigraph facts is obtained from asic instance.

Reply:
Removed